test/sandbox/test/rendering_test.rb

```diff
@@ -92,14 +92,12 @@ def test_raises_error_when_with_content_is_called_without_any_values
   def test_render_without_template
     render_inline(InlineComponent.new)
 
-    assert_predicate InlineComponent, :compiled?
```

Comment:
Why take these assertions out?

Reply:
I thought it wasn't prudent to reference internal APIs in tests, but IMO thats not a strong enough reason to modify tests.